ci/jenkins/test.sh

```diff
@@ -977,6 +994,14 @@ EOF
     done
 }
 
+function configure_kind_ipam_routes {
+    local kind_node="$1"
+
+    local pod_cidr=$(kubectl get nodes "${kind_node}" -o yaml | grep podCIDR | awk 'FNR == 1 {print$2}')
+    local node_ip=$(kubectl get nodes "${kind_node}" -o wide | awk 'FNR == 2 {print$6}')
+    sudo ip route add "$pod_cidr" via "$node_ip"
+}
+
 export KUBECONFIG=${KUBECONFIG_PATH}
 if [[ $TESTBED_TYPE == "flexible-ipam" ]]; then
     ./hack/generate-manifest.sh --flexible-ipam --multicast --verbose-log > build/yamls/antrea.yml
```

Comment:
Please re-base your codes, this part for manifest generation has been changed. Please use the new way to generate manifest.